### PR TITLE
{math}[GCCcore/11.3.0] libcerf v2.1

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
himem01.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/89f102bff1a5bc5b8c9aff33cbf555d8 for a full test report.

File: easybuild/easyconfigs/l/libcerf/libcerf-2.1-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+toolchainopts = {'pic': True}
+
+source_urls = ['https://jugit.fz-juelich.de/mlz/libcerf/-/archive/v%(version)s/']
+sources = ['libcerf-v%(version)s.tar.gz']
+checksums = ['8a1cd8b7fae04b82a95168252129b8c1baca098a285ff8d3f25781dead14b75a']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+    ('Perl', '5.34.1'),  # required for pod2html
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/libcerf/libcerf-2.1-GCCcore-11.3.0.eb:24]: the `-minimal` version should suffice here, it provides `pod2html`
```suggestion
    ('Perl', '5.34.1', '-minimal'),  # required for pod2html
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-ubuntu-eb - Linux Ubuntu 22.04, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.4
See https://gist.github.com/e661a2febc60a6e399ae7c6484eaaa22 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bdw-opensuse-154 - Linux openSUSE Leap 15.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 510.85.02, Python 3.6.15
See https://gist.github.com/9f31e96b8c7917a0f4e3ee636e120b5a for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**